src/LiveComponent/assets/src/live_controller.ts

```diff
@@ -195,20 +189,8 @@ export default class extends Controller {
         this._makeRequest(null);
     }
 
-    _getValueFromElement(element: HTMLElement){
-        const value = element.dataset.value || element.value;
-
-        if (!value) {
-            const clonedElement = (element.cloneNode());
-            // helps typescript know this is an HTMLElement
-            if (!(clonedElement instanceof HTMLElement)) {
-                throw new Error('cloneNode() produced incorrect type');
-            }
-
-            throw new Error(`The update() method could not be called for "${clonedElement.outerHTML}": the element must either have a "data-value" or "value" attribute set.`);
-        }
```

Comment:
Unrelated bug fix from a feature I merged recently #203. That PR added this check, but it IS ok to have an empty value - it just means, for example, that a textarea is empty! For some reason, the tests didn't catch this before, but after this change, one test DID start failing.